packages/tasit-account/dist/ethers.test.js

```diff
@@ -0,0 +1,136 @@
+"use strict";
```

Comment:
Having the dist files around does make it more confusing to review PRs. But I don't see a good way around that

Reply:
@marcelomorgado 
I'm thinking the flow proposed here could work. So we may be able to remove dist files from source control after all and just keep a hash of the latest dist files.
https://twitter.com/paulcowgill/status/1074713318416355328

If you agree, we can create an issue for this.

Reply:
I agree, great solution!